dotcom-rendering/package.json

```diff
@@ -175,47 +184,59 @@
 		"eslint-stats": "^1.0.1",
 		"execa": "^5.0.0",
 		"express": "^4.17.3",
+		"express-serve-static-core": "^0.1.1",
 		"fetch-mock": "^9.11.0",
 		"find": "^0.3.0",
-		"glob": "^8.0.3",
+		"form-data": "^4.0.0",
+		"glob": "^8.1.0",
 		"he": "^1.2.0",
 		"html-minifier": "^4.0.0",
+		"htmlparser2": "^8.0.1",
 		"inquirer": "^8.2.4",
 		"jest": "^24.9.0",
 		"jest-environment-jsdom-sixteen": "^1.0.3",
 		"jest-teamcity-reporter": "^0.9.0",
 		"jsdom": "^20.0.0",
 		"lint-staged": "^12.3.4",
 		"load-json-file": "^6.2.0",
+		"lodash": "^4.17.21",
 		"lodash.debounce": "^4.0.8",
 		"lodash.get": "^4.4.2",
```

Comment:
Gotta love the fact that we now have two flavours of `lodash`!

Reply:
Three even 🤯

Reply:
Well the `lodash.*` are tree-shaken ESM versions of the main one… so in my mind they are the same “flavour”.

Reply:
This actually caused an issue, which is fixed by #8761